### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by bradleybrand** [RGUnity/redguard-unity#12] [Q&A]
Is this project still currently being worked on? Many are still really interested. I have Redguard on GOG.com.

**Comment by Thane5** [comment]
Hi, yes, we are still working on this. 
Progress with the file converters has been quite slow unfortunately, but since the start of 2023, we managed to get large parts of two game systems done: the player inventory, and the save system.
I do occasionally post a screenshot or video on twitter if you're interested.

**Reply by bradleybrand** [reply]
Thanks for the reply. I would be interested in the project's Twitter. We saw Daggerfall Unity and wondered if there was anything ongoing for Redguard. I found online a mod manager for Redguard, with Unofficial Patch, but was unable to get it working. The DOS days are way passed. I appreciate your team's hard work.

**Reply by Thane5** [reply]
Here's the Twitter account: https://twitter.com/RGUnity
